index.test.js

```diff
@@ -0,0 +1,40 @@
+import { addString, toNumber, filterOrange } from "./index";
+
+
+//Q1
+
+describe("addString", () => {
+    test('adds "a" + "b" to equal "ab"', () => {
+        expect(addString("a", "b")).toBe("ab");
+    });
+
+    test('adds "a" + 1 to equal a1', () => {
+        expect(addString("a", 1)).toBe("a1");
+    });
+
+    test('adds 1 + 1 to equal -1', () => {
+        expect(addString(1, 1)).toBe(-1);
+    });
+})
+
+
+//Q2
+
+describe("toNumber", () => {
+    test('converts "123" to equal 123', () => {
+        expect(toNumber("123", 10)).toBe(123);
+    });
+
+    test('converts "7日" to equal 7', () => {
+        expect(toNumber("7日", 10)).toBe(7);
+    });
+})
+
+
+//Q3
+
+const fruit = ["orange", "apple", "peach"];
+
+test('Check if the array contains orange', () => {
+    expect(filterOrange(fruit)).toContain("orange");
+});
```

Comment:
add test not include, and
Testing when another type is passed.

Reply:
thank you for the review!
I've added a test. Is the intention of the review correct?
（I have looked into the different types of matchers, but I am not confident in my selections.）
3dd767e